packages/dashboard/src/views/Watchlists.js

```diff
@@ -21,60 +20,45 @@ import {
 import List from '../components/lists/List'
 import PageTitle from '../components/common/PageTitle'
 
-
-const Watchlists = ({
-  getWatchlist,
-  watchlist,
-  user,
-  isLoggedIn
-}) => {
-
+const Watchlists = ({ getWatchlist, watchlist, user, isLoggedIn }) => {
   useEffect(() => {
     if (isLoggedIn) {
-      getWatchlist({tokens: JSON.stringify(user.tokens)})
+      getWatchlist({ tokens: JSON.stringify(user.tokens) })
     }
   }, [isLoggedIn])
 
-  const checkWatchlist = () => {
-    console.log(watchlist)
+  const renderList = () => {
+    return watchlist.securities ? (
+      <List listTitle={'WST Watchlist'} items={watchlist.securities} />
+    ) : (
+      <h2 style={{ margin: 'auto' }}>Loading Watchlist Data...</h2>
+    )
   }
```

Comment:
If you dont need to do any logic beforehand in the function, arrow functions let you simply return whatever they point to.
```suggestion
  const renderList = () =>
    watchlist.securities ? (
      <List listTitle={'WST Watchlist'} items={watchlist.securities} />
    ) : (
      <h2 style={{ margin: 'auto' }}>Loading Watchlist Data...</h2>
    )
```